securemail/securemail.php

```diff
@@ -6,88 +6,104 @@
  * Author: Fabio Comuni <http://kirgroup.com/profile/fabrixxm>
  */
 
-require_once "include/Emailer.php";
+require_once 'include/Emailer.php';
 
 /* because the fraking openpgp-php is in composer, require libs in composer
  * and then don't use autoloader to load classes... */
-$path = __DIR__."/vendor/phpseclib/phpseclib/phpseclib/";
+$path = __DIR__ . '/vendor/phpseclib/phpseclib/phpseclib/';
 set_include_path(get_include_path() . PATH_SEPARATOR . $path);
 /* so, we don't use the autoloader and include what we need */
-$path = __DIR__."/vendor/singpolyma/openpgp-php/lib";
+$path = __DIR__ . '/vendor/singpolyma/openpgp-php/lib';
 set_include_path(get_include_path() . PATH_SEPARATOR . $path);
 
-require_once "openpgp.php";
-require_once "openpgp_crypt_symmetric.php";
+require_once 'openpgp.php';
+require_once 'openpgp_crypt_symmetric.php';
 
 
 function securemail_install() {
     register_hook('plugin_settings', 'addon/securemail/securemail.php', 'securemail_settings');
     register_hook('plugin_settings_post', 'addon/securemail/securemail.php', 'securemail_settings_post');
 
-    register_hook('emailer_send_prepare',  'addon/securemail/securemail.php', 'securemail_emailer_send_prepare');
+    register_hook('emailer_send_prepare', 'addon/securemail/securemail.php', 'securemail_emailer_send_prepare');
 
-    logger("installed securemail");
+    logger('installed securemail');
 }
 
 function securemail_uninstall() {
     unregister_hook('plugin_settings', 'addon/securemail/securemail.php', 'securemail_settings');
     unregister_hook('plugin_settings_post', 'addon/securemail/securemail.php', 'securemail_settings_post');
 
-    unregister_hook('emailer_send_prepare',  'addon/securemail/securemail.php', 'securemail_emailer_send_prepare');
+    unregister_hook('emailer_send_prepare', 'addon/securemail/securemail.php', 'securemail_emailer_send_prepare');
 
-    logger("removed securemail");
+    logger('removed securemail');
 }
 
-
-function securemail_settings(&$a,&$s){
-    if(! local_user()) {
+/**
+ * @brief Build user settings form
+ *
+ * @link https://github.com/friendica/friendica/blob/develop/doc/Plugins.md#plugin_settings 'plugin_settings' hook
+ *
+ * @param App $a App instance
+ * @param string $s output html
+ *
+ * @see App
+ */
+function securemail_settings(App &$a, string &$s){
```

Comment:
I'm sorry the `string` type hint is only available from PHP 7 onward, and we still have to support PHP5.